src/test/java/nl/dtl/fairmetadata4j/utils/MetadataUtilsTest.java

```diff
@@ -142,9 +142,25 @@ public void testParseNoTitleStatement() throws Exception {
         fail("This test is execpeted to throw an error");
     }    
     
-    public void testNoTitleWithFlag() {
-        
-    }
+    /**
+     * Test missing rdf statement but the mandatory check is disabled,
+     * so this test is excepted to pass
+     * 
+     * @throws Exception 
+     */
+    @Test
+    public void testParseNoTitleStatementWithFlag() throws Exception {
+        System.out.println("Test : Missing title statement");
```

Comment:
sysout vs using a logger, is there any reason for choosing the former over the latter?